alerting/src/test/kotlin/com/amazon/opendistroforelasticsearch/alerting/action/GetAlertsResponseTests.kt

```diff
@@ -96,7 +97,8 @@ class GetAlertsResponseTests : ESTestCase() {
                 null,
                 Collections.emptyList(),
                 "severity",
-                Collections.emptyList()
+                Collections.emptyList(),
```

Comment:
Could you align the rest of the Alert definition here and above so it all indented by four spaces?